user/inc/Arduino.h

```diff
@@ -4,6 +4,7 @@
 
 #define PARTICLE_ARDUINO_COMPATIBILITY
 #include "Particle.h"
+#include "spark_wiring_arduino.h"
```

Comment:
Changing my mind on how we include arduino support by default, but we still need it.  I'll follow up with a commit.